src/main/resources/static/css/custom.css

```diff
@@ -0,0 +1,15 @@
+/* Set a background image by replacing the URL below */
+body {
```

Comment:
jakbys tu body nazwal jakas nazwa typu class="costam" to moglibysmy tu dolozyc z reszty stron body, html { margin: 0 } 
wiesz o co chodzi? :P